test/dlc_tests/eks/eks_manifest_templates/tensorflow/inference/single_node_neuron_inference.yaml

```diff
@@ -0,0 +1,75 @@
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: <INFERENCE_SERVICE_NAME>
+  labels:
+    app: <INFERENCE_SERVICE_NAME>
+spec:
+  ports:
+  - name: http-tf-serving
+    port: 8500
+    targetPort: 8500
+  - name: grpc-tf-serving
+    port: 8501
+    targetPort: 8501
+  selector:
+    app: <INFERENCE_SERVICE_NAME>
+    role: master
+  type: ClusterIP
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: <INFERENCE_SERVICE_NAME>
+  labels:
+    app: <INFERENCE_SERVICE_NAME>
+    role: master
+spec:
+  replicas: <NUM_REPLICAS>
+  selector:
+    matchLabels:
+      app: <SELECTOR_NAME>
+      role: master
+  template:
+    metadata:
+      labels:
+        app: <SELECTOR_NAME>
+        role: master
+    spec:
+      serviceAccountName: dlc-serviceaccount
+      containers:
+      - name: tfs-service
+        image: <DOCKER_IMAGE_BUILD_ID>
+        command: ["/bin/sh","-c"]
+        args:
+        - "/usr/local/bin/entrypoint.sh"
+        ports:
+        - containerPort: 8500
+        - containerPort: 8501
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          capabilities:
+            add:
+              - IPC_LOCK
+        env:
+        - name: MODEL_NAME
+          value: <MODEL_NAME>
+        - name: MODEL_BASE_PATH
+          value: <MODEL_BASE_PATH>
+        - name: AWS_REGION
+          value: us-east-1
```

Comment:
Why is this set to `us-east-1` instead of `us-west-2`?

Since we have most of the tests already configured to be run on us-west-2, it would be easier to configure and debug tests in the same region rather than switching regions for the EKS Inference test, unless there's a strong reason to do so.